Script/char.dat

```diff
@@ -5178,8 +5898,15 @@ floatingeye
   DefaultMana = 0;
   CanOpen = false;
   TotalVolume = 30000;
-  HostileReplies == "@Dd stares at you.";
-  FriendlyReplies == "@Dd stares at you.";
+  HostileReplies =
+  {
+    4,
+    "@Dd stares at you intently.",
+    "@Dd blinks.",
```

Comment:
I'm not sure, but isn't it "blink" as an involuntary action, while you "wink" at somebody voluntarily? I can change it, I just don't know if one is inappropriate, or if I should just add a new line about winking while leaving blinking in.